src/material/dialog/_dialog-theme.scss

```diff
@@ -20,3 +20,21 @@
     @include mat-typography-level-to-styles($config, title);
   }
 }
+
+@mixin mat-dialog-density($density-scale) {}
```

Comment:
I'm wondering if we should omit the density mixin from non-experimental components for which we know there will be an MDC-based version

Reply:
I think it would rather make things more complicated if we add density mixins based on conditions. The way it is right now is that every Material Design based component provides the three theming system parts (`color`, `typography` and `density`). This makes it consistent.

I don't think it has any negative effect to have those empty mixins for the time being. Rather it seems more beneficial to provide all mixins consistently for every Material component